config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml

```diff
@@ -101,7 +101,7 @@ presubmits:
           privileged: true
 
 postsubmits:
-  kubernetes/cloud-provider-vpshere:
+  kubernetes/cloud-provider-vsphere:
```

Comment:
@akutz This postsubmit was not running because of this typo.

Reply:
Doh! Thank you @cjwagner !